topics/git/README.md

```diff
@@ -244,6 +247,10 @@ git checkout HEAD~1 -- /path/of/the/file
 
 <details>
 <summary>How to squash last two commits?</summary><br><b>
+
+There are two ways to squash commits:
+- Interactive rebasing: `git rebase -i HEAD~n`, where n is the number of commits to squash
+- Merging with squash option: `git merge --squash branch`
```

Comment:
This doesn't answer the original question of "How to squash the last two commits?"